cityehr-quick-start-guide/src/main/xslt/create-topic-html.xslt

```diff
@@ -25,15 +34,29 @@
         </xsl:call-template>
       </head>
       <body>
-        <div id="top-nav">
+        <nav id="top-nav">
```

Comment:
Why is this PR changing something unrelated to its purpose?

Reply:
@adamretter I hoped you wouldn't mind about this small improvement towards semantical markup, since I was working on the same part of HTML.

Reply:
It should really be in separate PR as it is not related to the changes in this PR. Next time please @plutonik-a.